src/main.rs

```diff
@@ -0,0 +1,150 @@
+use crate::postgres::Postgres;
+use actyxos_sdk::event_service::{EventService, Subscription};
+use anyhow::Result;
+use backtrace::Backtrace;
+use db::{Db, DbConnection, DbEvent};
+use futures::{future::FutureExt, stream::StreamExt};
+use futures_batch::ChunksTimeoutStreamExt;
+use serde::{Deserialize, Serialize};
+use std::{env, time::Duration};
+use structopt::StructOpt;
+use tokio_compat_02::FutureExt as FutureExt02;
+use tracing::*;
+
+mod db;
+mod postgres;
+
+#[derive(StructOpt, Debug, Serialize, Deserialize)]
+#[structopt(
+    about = "Insert events into a Postgres database.\nEnvironment variables:\n  PGPASSWORD: PostgreSQL password, if not provided with --password/-p\n  AX_EVENT_SERVICE_URI: URL to connect to the Event Service. Default: http://localhost:4454/api/"
+)]
+struct Opt {
+    #[structopt(long, short)]
+    from_start: bool,
+    #[structopt(
+        long,
+        short = "w",
+        about = "If not provided, read from the PGPASSWORD environment variable"
+    )]
```

Comment:
```suggestion
    #[structopt(
        long,
        short = "w",
        env = "PGPASSWORD",
        hide_env_values = true
    )]
```
https://docs.rs/structopt/0.3.21/structopt/#environment-variable-fallback

Reply:
thanks!